### PR TITLE
SW541807 : FTC1020:Rainier: bmcweb error logs for lamp test to

### PR DESCRIPTION
**Pull request by GiridhariKrishna**
debug as the function is not available yet.

### CODE REVIEW

File: redfish-core/lib/oem/ibm/lamp_test.hpp [hunk 1]
```diff
@@ -28,6 +28,11 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+	        if(ec.value() == 5)
+		{
```
Comment (santoshpuranik) [inline redfish-core/lib/oem/ibm/lamp_test.hpp:32]: Please fix formatting.

File: redfish-core/lib/oem/ibm/lamp_test.hpp [hunk 2]
```diff
@@ -28,6 +28,11 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+	        if(ec.value() == 5)
```
Comment (santoshpuranik) [inline redfish-core/lib/oem/ibm/lamp_test.hpp:31]: Have you tested this? Can you add a tested section?

**Review by santoshpuranik — CHANGES_REQUESTED** [review]

File: redfish-core/lib/oem/ibm/lamp_test.hpp [hunk 3]
```diff
@@ -28,12 +28,19 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+	        if(ec == boost::system::errc::host_unreachable)
+		{
+		    BMCWEB_LOG_DEBUG << "lamp test not available yet";
+		    std::cout << " lamp test not available";
+		    return;
+		}
+		std::cout << "#111";
```
Comment (gtmills) [inline redfish-core/lib/oem/ibm/lamp_test.hpp:37]: extra debug trace

File: redfish-core/lib/oem/ibm/lamp_test.hpp [hunk 4]
```diff
@@ -28,12 +28,19 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+	        if(ec == boost::system::errc::host_unreachable)
+		{
+		    BMCWEB_LOG_DEBUG << "lamp test not available yet";
+		    std::cout << " lamp test not available";
```
Comment (gtmills) [inline redfish-core/lib/oem/ibm/lamp_test.hpp:34]: we don't use cout

**Review by gtmills — CHANGES_REQUESTED** [review]
Needs cleaned up. I would like to see this get merged

File: redfish-core/lib/oem/ibm/lamp_test.hpp (hunk 4 above)
Comment (GiridhariKrishna) [reply redfish-core/lib/oem/ibm/lamp_test.hpp]: Yeah, sorry this patch came up unintentionally.

File: redfish-core/lib/oem/ibm/lamp_test.hpp (hunk 3 above)
Comment (GiridhariKrishna) [reply redfish-core/lib/oem/ibm/lamp_test.hpp]: Removed all of them

File: redfish-core/lib/oem/ibm/lamp_test.hpp [hunk 5]
```diff
@@ -28,6 +28,11 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+                if (ec.value() == 5) //generic:5 error
+                {
```
Comment (gtmills) [inline redfish-core/lib/oem/ibm/lamp_test.hpp:32]: Please review your code first. This is missing a }

**Review by gtmills — CHANGES_REQUESTED** [review]

File: redfish-core/lib/oem/ibm/lamp_test.hpp (hunk 5 above)
Comment (GiridhariKrishnan) [reply redfish-core/lib/oem/ibm/lamp_test.hpp:32]: "}" is  there below, I have some editor setting issues, it seems. I'll push it back once ready.

Comment (gtmills) [reply redfish-core/lib/oem/ibm/lamp_test.hpp:32]: Looks good, ignore my commit

**Review by gtmills — APPROVED** [review]

**Comment by gtmills** [comment]
For next time, please use a different branch name than [1020](https://github.com/GiridhariKrishna/bmcweb/tree/1020)

**Comment by gtmills** [comment]
e.g. a branch name like 
error-logs-for-lamp-test makes this more readable

**Review by santoshpuranik — APPROVED** [review]

**Pull request merged**